xain/grpc/coordinator.py

```diff
@@ -158,6 +177,28 @@ def Heartbeat(self, request, context):
         self.participants.update_expires(context.peer())
         return coordinator_pb2.HeartbeatReply()
 
+    def StartTraining(self, request, context):
+        print(f"Received: {type(request)} from {context.peer()}")
+        # NOTE self.theta is assumed to be initialised
```

Comment:
In this case `theta` shouldn't be an optional argument

Reply:
Following the suggestion in the [Hitchhiker's guide](https://docs.python-guide.org/writing/gotchas/), `theta` now has a sensible default of `[]` (without running into the "dangerous default" warning) so that `StartTraining` can be called safely.